content/topics/intro-to-relational-databases/_index.md

```diff
@@ -1,15 +1,99 @@
+
 ---
-title: SQL
-ready: True
+title: Introduction to relational database
+ready: true
 ---
 
-## Tutorial
 
-Complete the DataCamp Intro and Intermediate SQL courses:
+### INTRODUCTION TO RELATIONAL DATABASE (SQL)
+
+
+#### What is Relational DataBase? 
+
+Learn more here:[What is RDBMS](https://www.youtube.com/watch?v=h8IWmmxIyS0)
+
+- Introduction to SQL relational database [visit](https://www.mikedane.com/databases/sql/)
+
+- Basic SQL queries: [See these](https://www.mikedane.com/databases/sql/basic-queries/)
+
+
+#### LESSONS
+
+##### Lesson1:
+
+- Keys / Data and Tables:
+> [check these out](https://www.mikedane.com/databases/sql/tables-keys/)
+
+##### Lesson2:
+
+- Basics/Elements of SQL 
+> [check these out](https://www.mikedane.com/databases/sql/sql-basics/)
+
+##### Lesson3:
+  
+- SQL-Installation:
```

Comment:
No need to install. We use Docker. Just point them to the intro to docker topic (use a contentlink). 

Nobody should be running windows
Everyone should be able to get postgres up an running in a docker-composition